postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I didn't realise tailwind was preprocessed, I thought it was just a static stylesheet. Does this work with our current HMR logic?